requirements.txt

```diff
@@ -2,6 +2,7 @@
 GitPython
 Mako<2.0.0
 Sphinx
+aiohttp
```

Comment:
I think this can be dropped from `requirements.txt`, as `aiohttp` is explicitly listed as dependency for optional `async` feature.
For developer convenience we might introduce an additional `requirements-async.txt`(?) file containing `async` dependencies + reference to base `requirements.txt` (`-r requirements.utils.txt`)